test/SILGen/objc_direct.swift

```diff
@@ -47,14 +51,23 @@ markUsed(Bar.directClassMethod2())
 markUsed(bar.directProtocolMethod())
 // CHECK: function_ref @[[BYTE01]]-[Bar directProtocolMethod]
 
+// CHECK: sil [clang Bar.directProtocolMethod] @[[BYTE01]]-[Bar directProtocolMethod] : $@convention(objc_method)
```

Comment:
Maybe we should add a syntax for quoting SIL symbol names with non-identifier characters, like LLVM's `@"foo bar bas"` syntax, so that the printed SIL here can still round-trip.